README.md

```diff
@@ -98,15 +98,21 @@ If you need to run the backup job outside the schedule, use:
     tutor k8s backup
 
 The backup job backs up MySQL, MongoDB, and the Caddy data directory, 
-creates a tar file, and uploads it to an S3 storage bucket as set by 
-the `BACKUP_S3_*` configuration parameters. Note that the bucket must support 
-[object versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html).
+creates a tar file with a date stamp, and uploads it to an S3 storage bucket as 
+set by the `BACKUP_S3_*` configuration parameters. Note that the bucket must 
+support 
+[object versioning](https://docs.aws.amazon.com/AmazonS3/latest/userguide/Versioning.html)
+if you want to run multiple backups during the day.
 
 
-To restore from the latest version of the backup:
+To restore from the latest version of the backup made today:
 
     tutor k8s restore
 
+To restore from the latest version of the backup made on a particular date:
+
+    tutor k8s restore --date={YYYYMMDD}
```

Comment:
Any chance we could be ISO 8601 compliant here (i.e. `YYYY-MM-DD`, with hyphens)?

Reply:
Yes. I'll change that.